CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+<h2 align="center">Contributing</h2>
+
+<p align="center">Contributing to Dark Reader, can be in a variety of ways. Proceed reading the CONTRIBUTE.md to learn more about the variety of ways. Thank you in advance for reading this CONTRIBUTE.md.</p>
```

Comment:
Change this to `CONTRIBUTING.md` instead of `CONTRIBUTE.md`. Also, wrap `CONTRIBUTING.md` in `<code>` tags.